libs/ui-watt/src/lib/components/table/+storybook/storybook-table-overview.component.html

```diff
@@ -37,48 +23,31 @@
 >
 </mat-paginator>
```

Comment:
Agreed, it would make sense to use our implementation of Pagination to avoid usage of the mat-pagination directly :)

Reply:
@frontend-specialisten Our paginator is in DH, not in Watt. We can still use it but we should check whether that has an effect on "is-affected", bundle size, etc. when we import it.

Reply:
Oh my bad! Then I don’t think we should use it in watt as that wouldn’t make any sense for others than DH

Reply:
With the upcoming additions to the table (paginator as table footer), it will probably make sense to create a watt pagination component and then move the translation part of dh paginator into the dh lib `ui-watt-translation`. But that is definitely a task for later 🙂 

In the meantime I will add a comment about DH paginator, good point!

Reply:
3fa9a67292ba107d47f4f73e27e91a48c846d607